src/store/modules/events/index.ts

```diff
@@ -323,6 +325,62 @@ const module: Module<EventsModuleState, RootState> = {
       return recentEvents.dailyInfo.length !== RECENT_EVENTS_FETCH_LIMIT;
     },
 
+
+    /**
+     * Get data for displaying the number of errors for each day from a specific timestamp
+     * @param {function} commit - standard Vuex commit function
+     * @param {string} projectId - id of the project to fetch data
+     * @param {number} minTimestamp - timestamp from which we start taking errors
+     * @return {Promise<{timestamp: number, totalCount: number}[]>} - data for the chart
+     */
+    async [FETCH_CHART_DATA]({ commit }, { projectId, minTimestamp }): Promise<{timestamp: number, totalCount: number}[]> {
+      const chartData = await eventsApi.fetchChartData(projectId, minTimestamp);
+      const day = 86400000;
+      const now = Date.now();
+      const firstMidnight = new Date(minTimestamp * 1000).setHours(24, 0, 0, 0);
+
+      if (!chartData) {
+        return [];
+      }
+
+      /**
+       * Turning it into a chart format
+       */
+      let groupedData = Object.values(groupByGroupingTimestamp(chartData))
+      .reduce((acc: {timestamp: number, totalCount: number}[], val:any, i) => {
```

Comment:
Надо бы линтером пройтись

Reply:
Если тип элемента массива сложный, то лучше писать так
```suggestion
      .reduce((acc: Array<{timestamp: number, totalCount: number}>, val:any, i) => {
```

Reply:
От `any` надо избавиться, здесь же должен быть известен тип